templates/profile.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html.twig' %}
+
+{% block content %}
+    <main class="container">
```

Comment:
Please remove <Main> - it is already present in _base.html.twig_. According to html5 standards, there should not be more than 1 <main> element